scripts/variantstore/wdl/extract/create_vat_inputs.py

```diff
@@ -47,7 +47,7 @@ def download_ancestry_file(gcs_ancestry_file):
     blob = bucket.get_blob(blob_name)
     fd, temp_file = tempfile.mkstemp()
     # Close open descriptor, do not remove temporary file.
-    fd.close()
+    # fd.close()
```

Comment:
Miguel, can you help me figure this bit out?

Reply:
I would just delete the line if it's giving you problems, I didn't test this at all 🙈